docs/get-started/git/github-joomla-cms.md

```diff
@@ -0,0 +1,166 @@
+---
+sidebar_position: 4
+title: "GitHub Example 2: Joomla! CMS"
+---
+
+:::caution TODO
+
+This page is unfinished, but Cliff is working on it!
+
+:::
+
+## Getting Started
+
+If you would like to contribute to Joomla! core code you will need a [GitHub](https://github.com/) account. It is free and takes moments to set up. You will also need a working [LMWX]AMP stack on your local laptop or workstation. You will almost certainly need an IDE too! 
+
+## Fork joomla/joomla-cms on GitHub
+
+Log in to your GitHub account and type joomla/joomla-cms in the `Search or Jump to...` box at the top left. You need to be in the original joomla-cms repo.
+
+Click the `Fork` button at the top right of the screen. This will bring up a form asking you to confirm details of the fork you want to make. It is usually sufficient to select the green `Create fork` button.
+
+That will make a complete copy of the joomla-cms repo as it exists at this time in your own account. If someone updates the original later you can use the `Sync fork` button to bring your repo up to date.
+
+## Clone your Github fork locally
+
+The local Apache web server uses a specific folder for individual web sites. For example, on a Mac it may be /Users/username/Sites. You may have many sites for testing and developing different projects, each in a separate subfolder. The clone process will createe a folder within your current folder so first open a terminal window and switch to your sites folder:
```

Comment:
Typo fixed
```suggestion
The local Apache web server uses a specific folder for individual web sites. For example, on a Mac it may be /Users/username/Sites. You may have many sites for testing and developing different projects, each in a separate subfolder. The clone process will create a folder within your current folder so first open a terminal window and switch to your sites folder:
```